ql/models/model.hpp

```diff
@@ -121,7 +124,8 @@ namespace QuantLib {
         virtual void generateArguments() {}
         std::vector<Parameter> arguments_;
         boost::shared_ptr<Constraint> constraint_;
-        EndCriteria::Type shortRateEndCriteria_;
+        EndCriteria::Type endCriteria_;
```

Comment:
Ok about the values, but I'd keep the current name for `shortRateEndCriteria_` even though we all agree that it's unfortunate. It's a protected data member, so it could be used in user code.